torch/nn/functional.py

```diff
@@ -2495,7 +2495,7 @@ def _output_size(dim):
 
         # make scale_factor a tensor in tracing so constant doesn't get baked in
         if torch._C._get_tracing_state():
-            return [(torch.floor(input.size(i + 2) * torch.tensor(scale_factors[i]))) for i in range(dim)]
+            return [(torch.floor(input.size(i + 2) * torch.tensor(float(scale_factors[i])))) for i in range(dim)]
```

Comment:
Why is this necessary again? It looks quite benign, so if it ran in eager mode correctly, then it should run under the JIT as well I think?

Reply:
How do we get to the call of `floor_vml_cpu`?